src/app/payment/documentOnly/DocumentOnlyCustomForm.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo, FunctionComponent } from 'react';
+
+import { Fieldset } from '../../ui/form';
+import { CreditCardCustomField } from '../creditCard';
+
+export interface DocumentOnlyCustomFormProps {
+    additionalClassName: string;
+    autoComplete: string;
+    labelId: string;
+    name: string;
+}
+
+export interface DocumentOnlyCustomFormFieldsetValues {
+    ccDocument?: string;
+}
+
+const DocumentOnlyCustomForm: FunctionComponent<DocumentOnlyCustomFormProps> = ({
+    additionalClassName,
+    autoComplete,
+    labelId,
+    name,
+}) => (
+    <Fieldset>
+        <div className="form-ccFields">
```

Comment:
This also seems quite generic component, maybe it can be merged with `CreditCardCustomField` ? SingleTextFieldForm ?